pkg/kms/webkms/remotekms.go

```diff
@@ -217,9 +219,25 @@ func (r *RemoteKMS) doHTTPRequest(method, destination string, mReq []byte) (*htt
 //  - error if failure
 func (r *RemoteKMS) Create(kt kms.KeyType) (string, interface{}, error) {
 	startCreate := time.Now()
+
+	keyURL, _, err := r.createKey(kt, false)
+	if err != nil {
+		return "", nil, err
+	}
+
+	kid := keyURL[strings.LastIndex(keyURL, "/")+1:]
+
+	logger.Infof("overall Create key duration: %s", time.Since(startCreate))
```

Comment:
is this temporary?

Reply:
yes, we are going to clean up all those "debug" code closer to release